src/patterns/components/highlight-board/react/info/default.hbs

```diff
@@ -0,0 +1,183 @@
+<p class="sprk-u-mbm">See below for available customization options:</p>
+<table class="drizzle-c-Table drizzle-c-Table--wide sprk-u-mbm">
+  <colgroup>
+    <col class="sprk-u-Width-40">
+    <col>
+  </colgroup>
+
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">
+        children
+      </td>
+      <td>
+        Any renderable element(s)
+      </td>
+      <td>
+        Child elements are rendered in the content section of the component.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">
+        imgSrc
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        The source for the image. If this property is provided, imgAlt is required.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">
+        imgAlt
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        The alt text for the image. If this property is provided, imgSrc is required.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">
+        heading
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        The heading text.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">
+        ctaText
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        The text for the first CTA.
```

Comment:
We should define CTA here.

Reply:
replaced with "button". These CTAs will always render as a button, never a link.